rpcserver.go

```diff
@@ -1544,22 +1543,22 @@ func handleEstimateStakeDiff(s *rpcServer, cmd interface{}, closeChan <-chan str
 	// The expected stake difficulty. Average the number of fresh stake
 	// since the last retarget to get the number of tickets per block,
 	// then use that to estimate the next stake difficulty.
-	_, bestHeight := s.server.blockManager.chainState.Best()
-	lastAdjustment := (bestHeight / activeNetParams.StakeDiffWindowSize) *
+	best := s.server.blockManager.chain.BestSnapshot()
```

Comment:
Since `.Height` is the only field used, I would suggest minimizing changes here by just doing:
```Go
bestHeight := s.server.blockManager.chain.BestSnapshot().Height
```
That way nothing else has to change.

Minimal changes are preferable.